utils/date.h

```diff
@@ -5961,8 +5961,7 @@ parse(std::basic_istream<CharT, Traits>& is,
             }
             if (Y != not_a_year)
             {
-                if (!(static_cast<int64_t>(year::min()) <= Y &&
-                      Y <= static_cast<int64_t>(year::max())))
+                if (!std::in_range<int64_t>(Y))
```

Comment:
This will still always be true, as `Y` is a narrower type than `int64_t`, but fine I guess.